src/CloudKit/CKCompat.cs

```diff
@@ -29,14 +29,6 @@ public CKNotificationID ()
 #endif
 
 #if XAMCORE_2_0 || !MONOMAC
-	public partial class CKFetchNotificationChangesOperation {
-
-		// `init` does not work on watchOS but we can keep compatibility with a different init
-		public CKFetchNotificationChangesOperation () : this ((CKServerChangeToken) null)
```

Comment:
Won't this break code that's already using this constructor when that code runs on older watchOS versions - because on older watchOS you'll now call the broken `init` selector?